FordLabs/PeopleMover
# 153 people tag modal

## Issue
Resolves 153

## What was done
- [x] Made tags modal content more reusable
- [x] Added redux fields for people tags
- [x] Added and referenced modal for people tags

## How to test
1. Open a space with the #person-tags hash
2. Open person tags dropdown
3. Select "Add / edit"
4. Observe that the correct modal has opened
5. Add, edit, delete person tags
6. Open Add or Edit Person Form
7. Observe that the dropdown options for people tags in the form reflect the updated person tags based on your interactions in the previous modal
